libraries/fiff/fiff_named_matrix.h

```diff
@@ -316,7 +316,7 @@ inline bool operator== (const FiffNamedMatrix &a, const FiffNamedMatrix &b)
             a.ncol == b.ncol &&
             a.row_names == b.row_names &&
             a.col_names == b.col_names &&
-            a.data.isApprox(b.data, 0.0001f));
+            a.data.isApprox(b.data, 0.0001));
```

Comment:
why is the float qualifier removed?

Reply:
because data is an eigen matrix of type double not float. Compiler was throwing a warning.